File: forCS/FW2.0/NyMmd/jp/nyatla/nymmd/cs/types/PmdMaterial.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace jp.nyatla.nymmd.cs.types
{
    public class PmdMaterial
    {
        public MmdColor4 col4Diffuse = new MmdColor4();
        public MmdColor4 col4Specular = new MmdColor4();
        public MmdColor4 col4Ambient = new MmdColor4();
        public float fShininess;
        public String texture_name;
        public short[] indices;
    };
}

File: forJava/trunk/src/src/jp/nyatla/nymmd/types/PmdMaterial.java
package jp.nyatla.nymmd.types;


public class PmdMaterial
{
	public final MmdColor4 col4Diffuse=new MmdColor4();
	public final MmdColor4 col4Specular=new MmdColor4();
	public final MmdColor4 col4Ambient=new MmdColor4();
	public float fShininess;
	public String texture_name;
	public short[] indices;
	public int unknown;
};
